### PR TITLE
AN-78 Baseline creation

### PR DESCRIPTION
**Pull request by SimonEndres**

### CODE REVIEW

File: notebooks/evaluation/baseline_evaluation.ipynb [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "# create the references and baseline questions\n",
+    "refs = []\n",
+    "baseline_questions = []\n",
+    "for i, row in df_test.iterrows():\n",
+    "    refs.append((i, [row['Question 1'], '' if pd.isna(row['Question 2']) else row['Question 2'], '' if pd.isna(row['Question 3']) else row['Question 3']]))\n",
```
Comment (Niklas-23) [inline notebooks/evaluation/baseline_evaluation.ipynb:78]: You can use a list comprehension to do this, there is no need to add empty strings. 
`refs.append([row[col] for col in["Question 1", "Question 2", "Question 3"] if not pd.isna(row[col])])`

File: notebooks/evaluation/baseline_evaluation.ipynb [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
...ashing work?', '', '']), (421, ['How are infeasible test requirements handled in logic coverage criteria?', '', '']), (422, ['How is Base Choice Coverage defined?', '', '']), (423, ['What is an example for an SQL Injection Attack?', '', '']), (424, ['What is the concept of Critical Data?', '', '']), (425, ['What is a click fraud attack?', '', '']), (426, ['What is leadership according to famous personalities or movies?', '', '']), (427, ['Explain geometric interpretation of SVD.', '', '']), (428, ['Explain entropy.', 'What is Shannon entropy?', '']), (429, ['Which differences exist between genders (men and women) in leadership effectiveness?', '', '']), (430, ['What is Bitcoin and what is its goal and advantage?', '', '']), (431, ['What are the main differences between explicit and tacit knowledge?', '', '']), (432, ['What are general advices regarding potentially harmful e-mails?', '', '']), (433, ['How to recover when an agent has crashed?', '', '']), (434, ['How does the merge algorithm work?', '', '']), (435, ['How is the relevance score computed if relevance judgements are available?', '', '']), (436, ['What is the relationship between Bernoulli model and odds?', '', ''])]\n",
```
Comment (Niklas-23) [inline notebooks/evaluation/baseline_evaluation.ipynb]: Please use `df.head(5)` to print only a few lines and check if everything worked. There is no need to print everything here. This makes the notebook very unreadable

File: notebooks/evaluation/baseline_creation.ipynb
Comment (Niklas-23) [inline notebooks/evaluation/baseline_creation.ipynb:1]: I think this notebook is a bit unnecessary and we can do it all in one notebook. We do not need to overengineer it. Please use `df.head(5)` everywhere to print only a few lines. The notebook is currently not readable like this.

**Review by Niklas-23 — CHANGES_REQUESTED** [review]

File: notebooks/evaluation/baseline_evaluation.ipynb [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    }
+   ],
+   "source": [
+    "df = pd.read_csv(\"./../../datasets/baseline.csv\")\n",
+    "\n",
+    "df_train_val, df_test = train_test_split(df, test_size=0.2, random_state=42)\n",
+    "\n",
+    "df_test = df_test.reset_index(drop=True)\n",
+    "\n",
+    "display(df_test)"
```
Comment (Niklas-23) [inline notebooks/evaluation/baseline_evaluation.ipynb:60]: Same here, do not print the whole dataframe

File: notebooks/evaluation/baseline_evaluation.ipynb (hunk 1 above)
Comment (SimonEndres) [reply notebooks/evaluation/baseline_evaluation.ipynb:78]: I did it based on what Leon told me.

File: notebooks/evaluation/baseline_evaluation.ipynb (hunk 2 above)
Comment (SimonEndres) [reply notebooks/evaluation/baseline_evaluation.ipynb]: Cleared the output. Complete table was needed to scan through all results to find wrong labeled rows

File: notebooks/evaluation/baseline_creation.ipynb
Comment (SimonEndres) [reply notebooks/evaluation/baseline_creation.ipynb:1]: cleared the output

File: notebooks/evaluation/baseline_evaluation.ipynb (hunk 3 above)
Comment (SimonEndres) [reply notebooks/evaluation/baseline_evaluation.ipynb:60]: output cleared

**Review by TimBleicher — APPROVED** [review]

**Review by Niklas-23 — APPROVED** [review]
I still find two notebooks unnecessary, but otherwise it's ok like that

**Pull request merged**